### PR TITLE
Admin fixes

### PR DESCRIPTION
**Pull request by eluric**
Set ADMIN_CHANNEL_ID to be an environment variable
Added a command to clear the commands list. This should clear any old commands from the slash commands dropdown.
Added a command to change the hint channel. It's not a slash command so that it will not be visible to non-execs.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -62,4 +62,29 @@ async def sync(ctx: commands.context.Context):
...
+@commands.has_role(EXEC_ID)
+async def sethintchannel(ctx: commands.context.Context):
+    channel = ctx.channel
+
+    # set new channel id in environment variable
+    os.environ["ADMIN_CHANNEL_ID"] = str(channel.id)
+
+    # change the config variable
+    config["ADMIN_CHANNEL_ID"] = channel.id
+
```
Comment (SkellyBG) [inline main.py:86]: oh.. if you want to be able to modify the admin channel in code, this is not possible :c you'll need to make a table on postgres and store it. overwriting the environment variable only persist for the current container.

Comment (eluric) [reply main.py:86]: Yep, I know. In reality this command won't be needed since I don't foresee the hint channel needing to change.

**Comment by eluric** [comment]
Added a command that allows execs to delete puzzles.

Moved the commands for puzzle creation and deletion, as well as the command for setting the hint channel into a separate Admin cog. Discord allows you to hide certain slash commands from the dropdown list for users but this can only be applied to the entire cog. While these commands already only allowed execs to use them, they were still visible to regular users which was not ideal.

**Review by SkellyBG — APPROVED** [review]
nice! out of curiosity, which part of the code is responsible for only showing Admin cog to admin users?

**Comment by eluric** [comment]
> nice! out of curiosity, which part of the code is responsible for only showing Admin cog to admin users?

It's actually in Discord itself. Go to your server and then Server Settings > Integrations > HuntBot

**Pull request merged**